plugins/field-slider/test/field_slider_test.mocha.js

```diff
@@ -59,13 +58,23 @@ suite('FieldSlider', function() {
     assertSliderField(field, run.value, run.value, run.value, run.value);
   };
 
+
+  // TODO(https://github.com/google/blockly/issues/3903): Re-enable test cases
+  //  after fixing
+  invalidValueRuns[3].skip = true;
+
   runConstructorSuiteTests(
       FieldSlider, validValueRuns, invalidValueRuns, validRunAssertField,
       assertSliderFieldDefault);
 
   runFromJsonSuiteTests(FieldSlider, validValueRuns, invalidValueRuns,
       validRunAssertField, assertSliderFieldDefault);
 
+
+  // TODO(https://github.com/google/blockly/issues/3903): Remove skip=false
```

Comment:
Why do you have to reset `skip`? Is the object being used again?

Reply:
Yes, it is used in the setValueTests L89. I need to reset skip so that the test case is not skipped for setValue tests.